roles/engine-perun/tasks/Debian.yml

```diff
@@ -115,30 +115,6 @@
     owner: "{{ perun_login }}"
     group: "{{ perun_group }}"
 
-- name: "Delete /var/log/perun-engine directory if exists"
-  file:
-    path: "/var/log/perun-engine"
-    state: absent
-
-- name: Create /var/log/perun-engine directory
-  file:
-    path: "/var/log/perun-engine"
-    state: directory
-    owner: "{{ perun_login }}"
-    group: "{{ perun_group }}"
-
-- name: "Delete /var/log/perun-engine/perun-engine-out.log file if exists"
```

Comment:
I think deletion/creation of ``/var/log/perun/perun-engine-out.log`` should stay, since init.d script doesn't create it, but wants to append stdout/stderr for engine (note changing perun-engine folder to perun).